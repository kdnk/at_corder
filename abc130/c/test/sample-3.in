1010 23 505 11
