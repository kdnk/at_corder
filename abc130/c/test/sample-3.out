11615.000000 0
